docs/design/proposals/userID-mapping.md

```diff
@@ -0,0 +1,66 @@
+# UserID Mapping
+
+CephCSI mounts RBD or CephFS volumes on nodes using specific ceph userID. Currently,
```

Comment:
Please always write **Ceph-CSI**.

Uhm, a _ceph userID_? My first impression was that `root` is used for mounting... `userID` can be anything, and is commonly the _UID_ of the system user. Here you mean a Ceph user, which usually isn't a numeric ID.

Maybe phrase it differently, so that there can be no confusion.

> Ceph-CSI connects to the Ceph cluster with the credentials of a Ceph user account. The account is used to communicate with the MONs, OSDs and other services to attach RBD-images and mount CephFS volumes.